cves/CVE-2004-0493.yml

```diff
@@ -118,8 +121,8 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
+    - commit: ce0a2e4d24640502512311171818edce7edc55f3
```

Comment:
Where did you get this hash?  I wasn't able to find the associated commit on either the httpd GitHub or from the cloned repo from nitron.

Reply:
I was able to find it in the github repo

Reply:
Oh okay, weird.  Must have just overlooked it then!